lms/static/js/student_account/views/LoginView.js

```diff
@@ -105,10 +106,19 @@
                             anchorEnd: HtmlUtils.HTML('</a>')
                         });
 
-                    if (this.resetMessage.find('p').length === 0) {
-                        this.resetMessage.append(HtmlUtils.joinHtml(successMessage).toString());
-                    }
+                /* Temporarily remove these, then re-add them after the message
+                 * container has been shown. This seems to make them more visible
+                 * to screen readers in certain browsers (like IE 11).
+                 */
+                    $title.remove();
+                    $msg.remove();
                     this.element.show(this.$resetSuccess);
+                    this.$resetSuccess.append($title);
+                    this.$resetSuccess.append($msg);
+
+                    if ($msg.find('p').length === 0) {
+                        $msg.append(HtmlUtils.joinHtml(successMessage).toString());
+                    }
```

Comment:
We shouldn't be rendering the error title and message by default if we are always removing them before display. The code should be updated to load the error container without them and generate the HTML and append it before displaying any errors.